pkg/netpol/eval/internal/k8s/netpol.go

```diff
@@ -112,8 +115,16 @@ func (np *NetworkPolicy) getPortsRange(rulePort netv1.NetworkPolicyPort, dst Pee
 	return start, end, portName, nil
 }
 
+const (
+	minimumPort = 1
+	maximumPort = 65535
```

Comment:
ports are already defined in `pkg/netpol/internal/common/portset.go` (can be exported and used from there)

Reply:
done